src/plays/image-gallery-using-unsplash-api/styles.css

```diff
@@ -0,0 +1,112 @@
+/* enter stlyes here */
+.unsplash-image-gallery-body {
+  padding: 0;
+  margin: 0;
+  font-family: 'IBM Plex Sans', sans-serif;
+  background-color: whitesmoke;
+}
+
+.searchedPhotos > div > a > img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  border-radius: 5px;
+}
+
+.searchedPhotos {
+  display: grid;
+  margin: 2rem;
+  grid-gap: 10px;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-auto-rows: 600px;
+  grid-auto-flow: dense;
+}
+
+img {
```

Comment:
No tag selector override plz.

Reply:
![image](https://user-images.githubusercontent.com/94108842/220402899-d5d37567-4c7b-4b6b-91c6-b7972844c068.png)
Hello Tapas, 

I have updated the styles.css folder and the code is now working fine without any errors.

Regards